### ISSUE TITLE
cannot build with OpenSSL 1.1.1a

### ISSUE THREAD

**Issue by tycho** [dani-garcia/bitwarden_rs#337]
Building from latest tree, commit f571df7367991d804603630e04a36d5d96feac11. Ran into this:

```
error: failed to run custom build command for `openssl v0.9.24`
process didn't exit successfully: `/build/bitwarden_rs-git/src/bitwarden_rs/target/release/build/openssl-554be44989908808/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Unable to detect OpenSSL version', /build/.cargo/registry/src/github.com-1ecc6299db9ec823/openssl-0.9.24/build.rs:16:14
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```

I have OpenSSL 1.1.1a installed. This is apparently a known issue with rust-openssl 0.9.24: https://github.com/sfackler/rust-openssl/issues/994

And there are subsequent releases of rust-openssl which (theoretically) work with the OpenSSL 1.1.1a host libraries. 

Any way to fix this dependency chain to pull in the newer version? It looks like 0.9.24 is being pulled in by native-tls 0.1.5 (which also has newer versions available).

**Comment by tycho** [comment]
Thanks for the fast response! Your suggestion worked perfectly.

**Comment by dani-garcia** [comment]
The PR has been merged, so this should be fixed with 60f6a350be1e
